Frontend-user-interface/src/pages/GameList.js

```diff
@@ -49,12 +50,12 @@ const GameList = (props) => {
             if (user) {
                 const uid = user.uid;
                 fetchGames(uid);
-
             } else {
                 redirect('/login');
             }
         });
-    }, []);
+        setGameAdded(false);
+    }, [gameAdded]); 
```

Comment:
Is line 57 not redundant given that gameAdded is initiated to false when the component is rendered on line 19 ?

Reply:
No, I don't think it is redundant. The component must be rendered every time its state changes but is only initialyzed line 19. setGameAdded is called also called line 67, which trigger the rendering and, then, change the state back to false.